MapboxNavigation/DayStyle.swift

```diff
@@ -25,7 +25,6 @@ extension UIColor {
     class var trafficModerate: UIColor { get { return #colorLiteral(red: 1, green: 0.5843137255, blue: 0, alpha: 1) } }
     class var trafficHeavy: UIColor { get { return #colorLiteral(red: 1, green: 0.3019607843, blue: 0.3019607843, alpha: 1) } }
     class var trafficSevere: UIColor { get { return #colorLiteral(red: 0.5607843137, green: 0.1411764706, blue: 0.2784313725, alpha: 1) } }
-    class var trafficAlternateLow: UIColor { get { return #colorLiteral(red: 0.4666666687, green: 0.7647058964, blue: 0.2666666806, alpha: 1) } }
```

Comment:
`trafficAlternateLow` is not used anywhere, so probably can be removed.

Reply:
Looks like any usage of it was removed in #564. However, removing it would be a backwards-incompatible change, no? Perhaps we should only deprecate it for now.

Reply:
As I can see it's part of `UIColor` extension, which is not public, so there are no end users who can rely on it. I think there is no need in deprecation and removal is safe in this case.